indexes/private.js

```diff
@@ -110,46 +106,43 @@ module.exports = function (dir, keys) {
     return { offset: record.offset, value: buf }
   }
 
+  const bValue = Buffer.from('value')
+  const bContent = Buffer.from('content')
+
   function decrypt(record, streaming) {
```

Comment:
This diff might be hard to read, but essentially it's just flattening the nested `if`s. Plus, `record.value` => `recBuffer`.